example/run_demo.py

```diff
@@ -93,6 +93,16 @@ def initialize_rpc_server(
     log.info(rpc_subprocess.args)
 
     app = KaiRpcApplication()
+    log.setLevel(TRACE)
```

Comment:
is this needed?

Reply:
Nope, this was just for testing. Removing.